src/massive/munit/client/HTTPClient.hx

```diff
@@ -291,7 +291,7 @@ class URLRequest
 		#if (js || neko || cpp || java || cs || python || php)
 		client.onData = onData;
 		client.onError = onError;
-			#if js
+			#if(js && !nodejs)
```

Comment:
Why adding `-D nodejs` when you have `hxnodejs` flag already?

Reply:
-lib hxnodejs

:) but you can use `-lib nodejs` and others